src/tweenjs/Ease.js

```diff
@@ -42,11 +42,11 @@ this.createjs = this.createjs||{};
 	 *
 	 * Most methods on Ease can be passed directly as easing functions:
 	 *
-	 *      Tween.get(target).to({x:100}, 500, Ease.linear);
+	 *      Tween.get(target).to({x:100}, 500, createjs.Ease.linear);
```

Comment:
Any chance you can also change the `Tween.get` usages to `createjs.Tween.get` as well? Appreciate the catch!

Reply:
Oh my god! I must be going blind... I fixed it.
Cheers!